ttt/lib/ttt/board.ex

```diff
@@ -1,16 +1,19 @@
 defmodule TTT.Board do
+  @x_mark "X"
+  @o_mark "O"
+
   def empty_board(dimension) do
-    Enum.chunk(Enum.to_list(1..dimension*dimension), dimension)
+    rows(Enum.to_list(1..dimension*dimension))
   end
 
   def play_move(board, move, mark) do
-    board
-    |> replace_at(mark, matrix_row_index(move, board), matrix_column_index(move, board))
+    List.replace_at(board, move - 1, mark)
   end
 
-  def next_mark_to_play(board, marks) do
+  def next_mark_to_play(board) do
     board
-    |> get_move_count_per_mark(marks)
+    |> rows
+    |> get_move_count_per_mark
```

Comment:
Not really sold on this. 
I was expecting something along the lines of
```
board
|> count_marks
|> pick_fewest
```
which I can read and understand. The current methods feel a little noisy...